modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -92,7 +92,7 @@ def log_bad_central_mail_response(resp)
 
     scope :received_or_processing, -> { where status: RECEIVED_OR_PROCESSING }
     scope :completed, -> { where status: COMPLETE_STATUSES }
-    scope :has_pii, -> { where.not encrypted_form_data: nil, encrypted_auth_headers: nil }
+    scope :has_pii, -> { where(" encrypted_form_data != 'nil' OR encrypted_auth_headers != 'nil'") }
```

Comment:
Double check that the old code and the new are creating the same SQL statement. Remember that an empty value in ruby is `nil` but in SQL it's `null`.

Reply:
Ohhhh! Yes, I see! Multiple ways to solve. I'll push a new one up in a sec